edge-server/anthos/node-setup-common.sh.tmpl

```diff
@@ -137,91 +143,29 @@ setup_vlan_control_plane() {
-      cat "${SETUP_DIR}/edge-server/config-section.toml" >>"${CONTAINERD_TOML_PATH}"
-    else
-      echo "config.toml has already been configured with Nvidia runtime as the default one, skip..."
-    fi
-
-    ## Update to use Nvidia runtime.
-    ## Update default runtime to nvidia only if the Nvidia runtime configuration section exists.
-    if ! grep -q 'default_runtime_name = "nvidia"' "${CONTAINERD_TOML_PATH}" &&
-      grep -q 'plugins."io.containerd.grpc.v1.cri".containerd.runtimes.nvidia' "${CONTAINERD_TOML_PATH}"; then
-      echo "config.toml does not use Nvidia runtime, updating..."
-      sed -i 's/default_runtime_name = "runc"/default_runtime_name = "nvidia"/g' "${CONTAINERD_TOML_PATH}"
-    else
-      echo "config.toml already been updated with Nvidia default runtime, skip..."
-    fi
-
-    ## Restart containerd
-    echo "Restarting Containerd service..."
-    systemctl restart containerd
-    echo "Restarting Containerd service...Done"
-
-    # Install Nvidia Device Pligin
-    ## Wait untill containerd is running.
-    COUNTER=0
-
-    while [ $COUNTER -lt 30 ]; do
-      COUNTER=$((COUNTER + 1))
-      sleep 1s
-      if systemctl status containerd | grep -q "Active: active (running)"; then
-        echo "Containerd is in running state."
-        break
-      elif [ $COUNTER -lt 30 ]; then
-        echo "Containerd is not in running state, waiting..."
-      else
-        echo "Containerd is not in running state after ${COUNTER} seconds, exit..."
-        exit 1
-      fi
-    done
-
-    if kubectl get ds -n kube-system --kubeconfig="$SETUP_DIR/bmctl-workspace/$ANTHOS_MEMBERSHIP_NAME/$ANTHOS_MEMBERSHIP_NAME-kubeconfig" | grep -q "nvidia-device-plugin-daemonset"; then
-      echo "nvidia-device-plugin-daemonset exists, deleting"
-      kubectl delete ds nvidia-device-plugin-daemonset \
-        --kubeconfig="$SETUP_DIR/bmctl-workspace/$ANTHOS_MEMBERSHIP_NAME/$ANTHOS_MEMBERSHIP_NAME-kubeconfig" \
-        -n kube-system
-    fi
-    echo "Creating nvidia-device-plugin-daemonset"
-    kubectl apply -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v0.12.3/nvidia-device-plugin.yml \
-      --kubeconfig="$SETUP_DIR/bmctl-workspace/$ANTHOS_MEMBERSHIP_NAME/$ANTHOS_MEMBERSHIP_NAME-kubeconfig"
+  echo "[Info] KUBECONFIG=${KUBECONFIG}"
+  if [ -z "${1}" ]; then
+    echo "[Info] No path specified, use default"
+    USER_HOME_PATH="/home/viai-admin"
   else
-    echo "[Error]File:${CONTAINERD_TOML_PATH} not found, can not update ${CONTAINERD_TOML_PATH}"
+    USER_HOME_PATH="${1}"
+    echo "[Info] USER_HOME_PATH=${USER_HOME_PATH}"
+    shift
   fi
+  echo "[Info] KUBECONFIG=${KUBECONFIG}"
+  echo "[Info] Appending KUBECONFIG to ${USER_HOME_PATH}/.bashrc"
+  echo "export KUBECONFIG=${KUBECONFIG}" >>"${USER_HOME_PATH}/.bashrc"
+
+  echo "[Info] Appending KUBECONFIG to ${USER_HOME_PATH}/.profile"
+  echo "export KUBECONFIG=${KUBECONFIG}" >>"${USER_HOME_PATH}/.profile"
```

Comment:
What if `.bashrc` and `.profile` don't exist?

Reply:
They are created when a new user is created. The script is run after OS installation completed where we have user`ubuntu` and user `viai-admin` created. When the script is ran they should already be there.

The script either:

- List folders under `/home`, and iterate through each folders to create / append the file. Or
- Update files in '/home/viai-admin', so there is a chance somehow `viai-admin` was not created.
   * Added a check here to confirm wether the folder exists.

If the file was somehow not created, `>>` automatically create the file, so we are still good.